sources/packages/web/src/services/http/dto/User.dto.ts

```diff
@@ -0,0 +1,41 @@
+export interface BCeIDDetailsAPIOutDTO {
```

Comment:
I think all  `APIDTO`  should be `class`

Reply:
In the backend we are using class because of swagger but in the web side all dtos are interfaces.